src/components/interaction-designer/Block.vue

```diff
@@ -131,87 +132,80 @@
-                </plain-draggable>
-
-                <template v-if="isConnectionCreateActive && isExitActivatedForCreate(exit) && livePosition">
+            <div
+              class="block-exit-tag badge badge-warning d-flex justify-content-center"
+              :style="{background: exitBackgroundColor(exit)}"
+              @mouseenter="exitMouseEnter(exit)"
+              @mouseleave="exitMouseLeave(exit)">
+              <span
+                v-if="!(exitHovers[exit.uuid] || isExitActivatedForCreate(exit))"
+                v-b-tooltip.hover.top="exit.test"
+                class="block-exit-tag-text align-self-center">
+                {{ exit.name || '(untitled)' }}
+              </span>
+
+              <span class="align-self-center">
+                <template v-if="exit.destination_block == null">
+                  <plain-draggable
+                    v-if="exitHovers[exit.uuid] || isExitActivatedForCreate(exit)"
+                    :id="`exit/${exit.uuid}/pseudo-block-handle`"
+                    :key="`exit/${exit.uuid}/pseudo-block-handle`"
+                    v-b-tooltip.hover.top="transIf(isEditable, 'flow-builder.tooltip-new-connection')"
+                    class="btn btn-xs btn-flat p-0"
+                    :is-editable="isEditable"
+                    @initialized="handleDraggableInitializedFor(exit, $event)"
+                    @dragStarted="onCreateExitDragStarted($event, exit)"
+                    @dragged="onCreateExitDragged($event)"
+                    @dragEnded="onCreateExitDragEnded($event, exit)"
+                    @destroyed="handleDraggableDestroyedFor(exit)">
+                    <i class="glyphicon glyphicon-move" />
+                  </plain-draggable>
+
+                  <template v-if="isConnectionCreateActive && isExitActivatedForCreate(exit) && livePosition">
+                    <div
+                      :id="`exit/${exit.uuid}/handle`"
+                      class="btn btn-xs p-0">
+                      <i class="glyphicon glyphicon-move" />
+                    </div>
+                    <connection
+                      :key="`exit/${exit.uuid}/line-for-draft`"
+                      :repaint-cache-key-generator="generateConnectionLayoutKeyFor"
+                      :source="block"
+                      :target="blocksById[exit.destination_block]"
+                      :exit="exit"
+                      :position="livePosition" />
+                  </template>
+                </template>
+
+                <template v-if="exit.destination_block != null">
                   <div
                     :id="`exit/${exit.uuid}/handle`"
-                    class="block-handle-move btn btn-xs">
-                    <i class="glyphicon glyphicon-move" />
+                    :key="`exit/${exit.uuid}/handle`"
+                    class="btn btn-xs btn-flat">
+                    <font-awesome-icon
+                      v-if="exitHovers[exit.uuid]"
```

Comment:
No longer affected by isEditable? 🙀

Reply:
isEditable was removed on the master branch, I assumed it was removed to fix some bugs. But I notice that the icons come up on hover now. I figured a better way to deal with this by disabling all mouse events on the whole footer (might work on the root div of Block.vue too). Let me know your thoughts on this approach?